From 618034ccec2d669ea06e3e1ccd54b1580f47dc3c Mon Sep 17 00:00:00 2001
From: Chris Sewell <chrisj_sewell@hotmail.com>
Date: Mon, 27 Mar 2023 11:09:28 +0200
Subject: [PATCH] fix aiida-core dependency

---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 861f332..c733a9d 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -22,7 +22,7 @@ classifiers = [
 keywords = ['aiida', 'abinit']
 requires-python = '>=3.8'
 dependencies = [
-    'aiida_core[atomic_tools]~=1.6.3,<3.0.0',
+    'aiida_core[atomic_tools]>=1.6.3,<3.0.0',
     'aiida-pseudo~=0.6.1',
     'abipy>=0.8.0',
     'packaging',
-- 
2.37.3
